<ui version="4.0" >
 <class>QtMonkeyMainWin</class>
 <widget class="QWidget" name="QtMonkeyMainWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QtMonkey</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Path to testing application: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTestApp_" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbBrowse_" >
       <property name="text" >
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Arguments: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTestAppArgs_" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Script:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbNewScript_" >
       <property name="text" >
        <string>New Script</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbLoadScriptFromFile_" >
       <property name="text" >
        <string>Load From File</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSaveScriptToFile_" >
       <property name="text" >
        <string>Save To File</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="outputWindowsSplitter_" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTextEdit" name="teScriptEdit_" >
      <property name="lineWrapMode" >
       <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
     <widget class="QFrame" name="outputCtrlFrame_" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Output:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbClearLog_" >
           <property name="text" >
            <string>Clear Output</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>321</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTextEdit" name="teLog_" >
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QCheckBox" name="cbInsertEventsAtCursor_" >
         <property name="text" >
          <string>Insert Events at Text Cursor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbStartRecording_" >
         <property name="text" >
          <string>Start Event Record</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbRunScriptDebug_" >
       <property name="text" >
        <string>Run Script(Debug mode)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QCheckBox" name="cbProtocolRunning_" >
         <property name="text" >
          <string>Protocol Execution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbDemonstrationMode_" >
         <property name="text" >
          <string>Demonstration Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRunScript_" >
         <property name="text" >
          <string>Run Script</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
